contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -34,9 +34,7 @@ import "./DaoOperator.sol";
  *          Expiry timestamp for data is set. when data expires. Fee handler reads new data from DAO.
  */
 
-interface IKyberProxy is IKyberNetworkProxy, ISimpleKyberProxy {
-    function kyberNetwork() external view returns (address);
-}
+interface IKyberProxy is ISimpleKyberProxy, IKyberNetworkProxy { }
```

Comment:
Do we have a standard for empty code blocks? With or without spacing?

Reply:
saw few places it is `contract A ... {}`

Reply:
let me merge
and try to align those later.